docs/src/ToricVarieties/ToricDivisorClasses.md

```diff
@@ -22,37 +22,32 @@ Toric divisor classes are equivalence classes of Weil divisors modulo linear equ
 ToricDivisorClass(v::AbstractNormalToricVariety, coeffs::Vector{T}) where {T <: IntegerUnion}
 ```
 
-### Special constructors
+### Addition, subtraction and scalar multiplication
 
-Addition of toric divisor classes `tdc1` and `tdc2` (on the same toric variety) and
-scalar multiplication with `c` (it can be either valued in `Int64` or `fmpz`)
-is supported via `c * tdc1 + tdc2`. One can subtract them via `tdc1 - tdc2`.
+Toric divisor classes can be added and subtracted via the usual `+` and `-`
+operators. Moreover, multiplication by scalars from the left is supported
+for scalars which are integers or of type `fmpz`.
 
+### Special divisor classes
 
-### Equality
+```@docs
+trivial_divisor_class(v::AbstractNormalToricVariety)
+anticanonical_divisor_class(v::AbstractNormalToricVariety)
+canonical_divisor_class(v::AbstractNormalToricVariety)
+```
 
-Equality of two toric divisor classes `tdc1` and `tdc2` (on the same toric variety)
-is achieved by checking if their difference is a trivial class, i.e. the divisor class of
-a principal toric divisor. This is implemented via `tdc1 == tdc2`.
 
+## Properties
 
-## Properties of toric divisor classes
+Equality of toric divisor classes can be tested via `==`.
 
-To check if a toric divisor class `tdc` is trivial, one can invoke `is_trivial(tdc)`.
+To check if a toric divisor class is trivial, one can invoke `is_trivial`.
```

Comment:
A divisor class is trivial if a divisor representing the class is principal. To check this, we can use `is_principal` for toric divisors.

My expectation for a method `is_trivial` for a toric divisor $D = a_1 D_1 + ... + a_n D_n$ (D_i the torus invariant prime divisors) is to tell if all coefficients $a_i$ vanish. However, this is different from telling if D is a principal divisor.

Do you agree? Does that make sense?

Reply:
It does make sense, but I meant to add it as such a `@docs` block. I think it is fine.

Reply:
Then yes, this is because the `is_trivial` docs grew too large.

Still, we have both functionality and documentation on this (cf. `ToricDivisors.md`):

```
To check if a toric divisor is trivial, one can invoke `is_trivial`.
Internally, this executes the following method:
```@docs
is_principal(td::ToricDivisor)
```

However, I am (on second thought) not really happy with what `is_trivial` does for toric divisors. For divisor classes, this is ok. But for a toric divisor `is_trivial` should act differently. Let me think of a fix for that...

Reply:
What do you think of b035c256d66aa533f29a7d88c4affe65510624dd?

`is_principal` is documented (with @docs block) while `is_trivial` is only mentioned in the documentation.

Note that this seems related to https://github.com/oscar-system/Oscar.jl/issues/1471. 

If I recall correctly, I could not simply add an @ref for `is_trivial` because somewhere in `OSCAR` there was (at the time that I tried = several weeks ago) documentation for an `is_trivial` method, which led to some error... Maybe this is best addressed separately?